src/vs/workbench/contrib/positronNotebook/browser/notebookCells/CellLeftActionMenu.tsx

```diff
@@ -51,6 +51,7 @@ export function CellLeftActionMenu({ cell }: CellLeftActionMenuProps) {
 	const executionStatus = useObservedValue(cell.executionStatus);
 	const duration = useObservedValue(cell.lastExecutionDuration);
 	const lastRunEndTime = useObservedValue(cell.lastRunEndTime);
+	const hasError = useObservedValue(cell.hasError);
```

Comment:
Might be better to avoid tracking this state both here and in `NotebookCellWrapper` and making one a prop? We might be doing that in a couple places